src/components/Expandable.tsx

```diff
@@ -34,11 +34,9 @@ export const Expandable: React.FC<ExpandableProperties> = ({
         <h3 className='h4 o-expandable_label'>{header}</h3>
         <span className='o-expandable_link'>
           <span className='o-expandable_cue o-expandable_cue-open'>
-            <span className='u-visually-hidden-on-mobile'>Show</span>
```

Comment:
I think we want keep these labels even though they are not shown in the [CFPB Expandable](https://cfpb.github.io/design-system/components/expandables).  
 
In the wireframes for SBL, I do see the `Show`/`Hide` labels displayed, so maybe we make these either customizable or togglable via a flag?

Reply:
@contolini do you concur?

Reply:
They're being phased out of cf.gov but yeah it'd be awesome to stick them behind a flag so they can be shown if desired.

Reply:
@contolini yeah but then that would be a separate feature?

Reply:
Actually, nvm. If the icon spacing issue that I flagged above is caused by the presence of the show/hide text, I'd recommend just removing the text. It'd be bad for us to have to write custom CSS to accommodate the text when the Design System recommends removing it.